draft-irtf-cfrg-opaque.md

```diff
@@ -676,7 +680,7 @@ Output:
 
 Steps:
 1. private_key = HashToScalar(seed, dst="OPAQUE-HashToScalar")
-2. public_key = private_key * G
+2. public_key = ScalarBaseMult(private_key)
```

Comment:
`ScalarBaseMult` is not defined in this document, if I'm not mistaken

Reply:
It's in the VOPRF document, Section 2.1 (see the citation added in this PR, which also covers where HashToScalar is introduced.)